VultisigApp/VultisigApp/Chains/OneInchSwaps.swift

```diff
@@ -58,10 +61,19 @@ private extension OneInchSwaps {
                 }
             }
         }
-
-        let gasPrice = BigUInt(quote.tx.gasPrice) ?? BigUInt.zero
-        // sometimes the `gas` field in oneinch tx is 0
-        // when it is 0, we need to override it with defaultETHSwapGasUnit(600000)
+        
+        var gasPrice = BigUInt(quote.tx.gasPrice) ?? BigUInt.zero
+        if gasPrice == 0 {
```

Comment:
you can't do this , and you should not do this either , because it break the cross sign between IOS and Android